packages/select/src/FilterMultiSelect/components/MenuLayout/MenuLoadingSkeleton/MenuLoadingSkeleton.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from "classnames"
 import { MenuFooter } from "../MenuFooter"
 import styles from "./MenuLoadingSkeleton.module.scss"
 
-export const MenuLoadingSkeleton = (
+export const MenuLoadingSkeleton = () => (
```

Comment:
`export const MenuLoadingSkeleton: React.VFC = () => (`

Just to keep consistent with our other components?

Reply:
Yeah sure - I noticed that `.VFC` is deprecated in favour of `.FC` though - do we want to use that instead?

Reply:
Yeah we need to still use .VFC because of compatibility with react 16 _(something like that)_

Reply:
Ah cool, thanks Nat